src/components/app.tsx

```diff
@@ -179,6 +179,8 @@ export class App extends React.Component<AppProps, AppState> {
         this.codap = new Codap((initialState:any) => {
             // merge saved initial state into current state
             this.setState(initialState);
+            if (initialState && (initialState.runLength != null))
```

Comment:
Maybe use curly brackets around if statement body to prevent ambiguity if additions to body are needed in future

Reply:
Ha!  I actually agree with you on this one, but I was recently scolded on another CC project for not using curly brackets on cases like this.  I've always found curly brackets around single line if bodies to be less readable and excessively verbose (vertical screen real estate tends to be at a premium and excessive brackets gobble it up).  Glad I'm not the only one who thinks this way!